build/manifest.ps1

```diff
@@ -1,18 +1,30 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-#!/usr/bin/env pwsh
-#Requires -PSEdition Core
+<#
+    .SYNOPSIS
+        Provides the list of artifacts (Packages and Assemblies) generated by this repository.
+    
+    .PARAMETER OutputFormat
+        Specifies if the output of this script should be a hashtable with the artifacts
+        as strings with the absolute path (AbsolutePath) or FileInfo structures.
+#>
+param(
+    [ValidateSet('FileInfo','AbsolutePath')]
+    [string] $OutputFormat = 'FileInfo'
+);
+
 
 & "$PSScriptRoot/set-env.ps1"
 
-@{
+$artifacts = @{
     Packages = @(
         "Microsoft.Quantum.IQSharp.Core",
         "Microsoft.Quantum.IQSharp.ExecutionPathTracer",
         "Microsoft.Quantum.IQSharp.Jupyter",
         "Microsoft.Quantum.IQSharp"
-    );
+    ) | ForEach-Object { Join-Path $Env:NUGET_OUTDIR "$_.$Env:NUGET_VERSION.nupkg" };
```

Comment:
Similar comment as in the other repos about `.snupkg`, though that's not currently applicable to this repo since it doesn't produce any `.snupkg` artifacts.

Reply:
I think they do. Practically it doesn't matter as for nuget packages, as opposed to dlls, we submit for signing everything that we found in the NUGET_OUTDIR folder (which includes the .snupkgs). So, this is not really needed for signing.
We do use it to validate that the package is created on local builds. So for now, I'm just going to leave it like this.